fastlane/Fastfile

```diff
@@ -906,6 +928,34 @@ platform :ios do
   end
 end
 
+def extracted_release_notes_file_path(app:)
+  paths = {
+    wordpress: File.join(PROJECT_ROOT_FOLDER, 'WordPress', 'Resources', 'release_notes.txt'),
+    jetpack: File.join(PROJECT_ROOT_FOLDER, 'WordPress', 'Jetpack', 'Resources', 'release_notes.txt')
+  }
+
+  # Good coding would have us handle the key not being found here. But, given
+  # this is a service method for an automation lane, we'd rather have the
+  # process break with a nil value and a human notice it immediately, than an
+  # incorrect file being used and the issue possibly going unnoticed for
+  # longer.
+  paths[app]
+end
+
+def print_release_notes_reminder
+  message = <<~MSG
+    The extracted release notes for WordPress and Jetpack were based on the same source.
+    Don't forget to remove any item that doesn't apply to the respective app before editorialization.
+
+    You can find the extracted notes at:
+
+    - #{extracted_release_notes_file_path(app: :wordpress)}
+    - #{extracted_release_notes_file_path(app: :jetpack)}
+  MSG
+
+  message.lines.each { |l| UI.important(l.chomp) }
```

Comment:
Not sure if there's a better way to print a multi line message via Fastlane. If I had put all the lines in the same `UI.important` call, only the first would have had the timestamp on screen, resulting in something that didn't look homogeneous to me:

![image](https://user-images.githubusercontent.com/1218433/149174328-3b869f7b-e81d-48d9-b05a-e73bf68b37a0.png)

Reply:
An cheap alternative would be to add an `\n` at the start of the message I guess, so that the line with the timestamp would be empty and all the remaining lines would all be aligned left? 🤷‍♂️ 

Both cases work for me, so I don't mind either way. Only benefit of this `\n`-prepending trick would be to make it easier to copy/paste (e.g. in case we want to post the text in Slack to as the Jetpack team), but at that point we'll likely edit the `.txt` files in VSCode anyway, so not sure it's such a benefit anyway.

Reply:
> Both cases work for me, so I don't mind either way. 

In that case, I'd like to keep it the way it is.